docusaurus/docs/publish-a-plugin/distribute-a-plugin.md

```diff
@@ -1,27 +0,0 @@
----
-id: distribute-a-plugin
-title: Distribute a plugin
-sidebar_position: 1
----
-
-To distribute a Grafana plugin either to the community or privately, the plugin must be signed so the Grafana application can verify its authenticity. You can do this with the `@grafana/sign-plugin` package.
-
-:::info
-
-It's not necessary to sign a plugin during development. The [Docker development environment](../get-started/set-up-development-environment.mdx) that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature.
```

Comment:
We've lost this information. We should add to the dev environment doc and maybe sign a plugin. WDYT @josmperez ?

Reply:
Good idea. Added to both places.